source/tutorials/qiime2-for-experienced-microbiome-researchers.rst

```diff
@@ -0,0 +1,296 @@
+
+Quality filtering
+'''''''''''''''''
+
+You can perform different types of quality filtering with the :doc:`quality filter <../plugins/available/quality-filter/index>` plugin.
+**TODO: what's the difference between the q-score and q-score-joined? Why are there two separate functions?**
+The option descriptions for each method cover the different types of available quality filtering.
+
+Dereplicating sequences
+'''''''''''''''''''''''
+
+No matter which type of clustering you do, you first need to dereplicate your sequences. The :doc:`q2-vsearch <../plugins/available/vsearch/index>` plugin's method :doc:`dereplicate-sequences  <../plugins/available/vsearch/dereplicate-sequences/>` performs this step.
+
+de novo clustering
+''''''''''''''''''
+
+Sequences can be clustered *de novo* based on their genetic similarity alone (i.e. with VSEARCH) or based on a combination of their genetic similarity and abundance distributions (i.e. with distribution-based clustering).
+
+-  **Similarity-based clustering.** The QIIME 2 VSEARCH plugin method :doc:`cluster-features-de-novo <../plugins/available/vsearch/cluster-features-de-novo/>` clusters OTUs. You can change the genetic similarity threshold with the ``--p-perc-identity`` parameter. The plugin wraps the VSEARCH ``--cluster_size`` function.
+-  **Distribution-based clustering** incorporates the similarity between sequences and their abundance distribution to identify ecologically meaningful populations. You can learn more about this method in the `plugin documentation <https://github.com/cduvallet/q2-dbotu>`__, `original paper <http://dx.doi.org/10.1128/AEM.00342-13>`__, and the `re-implementation update paper <https://doi.org/10.1371/journal.pone.0176335>`__. The ``call-otus`` function in the `q2-dbotu <https://github.com/cduvallet/q2-dbotu>`__ plugin performs distribution-based clustering on input data.
+
+Both of these functions take as input the output of ``q2-vsearch dereplicate-sequences``, which are dereplicated sequences with QIIME 2 data type ``'FeatureData[Sequence]'``, and a table of counts with QIIME 2 data type ``'FeatureTable[Frequency]'``.
+
+closed reference clustering
+'''''''''''''''''''''''''''
+
+Closed reference clustering groups sequences together which match the same reference sequence in a database with a certain similarity.
+
+VSEARCH can do closed reference clustering with the :doc:`cluster-features-closed-reference <../plugins/available/vsearch/cluster-features-closed-reference/>` method.
+This method wraps the ``--usearch_global`` VSEARCH function.
+You can decide which reference database to cluster against with the ``--i-reference-sequences`` flag.
+The input file to this flag should be a ``.qza`` file containing a fasta file with the sequences to use as references, with QIIME 2 data type ``FeatureData[Sequence]``.
+Most people use GreenGenes or SILVA, but others curate their own databases or use other standard references (e.g. UNITE for ITS data).
```

Comment:
Most people use GreenGenes or SILVA -> Most people use GreenGenes or SILVA for 16S rRNA gene sequences